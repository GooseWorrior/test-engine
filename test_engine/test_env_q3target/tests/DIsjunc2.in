a||b
a
b
ab

q
d(a|b)|c
a
b
d
da
db
c
a(b|c)d
abd
acd
a
b
c
d
abcd
q
(a|b)(c|d)
a
b
c
d
ac
ad
bc
bd
abcd
q
(a|b)|(c|d)
a
b
c
d
ac
ad
bc
bd
abcd
q

