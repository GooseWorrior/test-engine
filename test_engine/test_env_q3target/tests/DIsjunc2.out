Disjunction (a,Disjunction (,b))
true
true
false
true
Disjunction (Concatenation (d,Disjunction (a,b)),c)
false
false
false
true
true
true
false
false
false
false
false
true
false
false
Concatenation (Disjunction (a,b),Disjunction (c,d))
false
false
false
false
true
true
true
true
false
Disjunction (Disjunction (a,b),Disjunction (c,d))
true
true
true
true
false
false
false
false
false

